File: task_d_evolve_api/notification_service.py
class NotificationService:
    def __init__(self):
        self.sent_notifications = []

    def send(self, user: str, message: str):
        """Send a notification to a user."""
        notification = f"To {user}: {message}"
        self.sent_notifications.append(notification)
        print(notification)

    def get_history(self) -> list:
        """Return the list of all sent notifications."""
        return self.sent_notifications.copy()

File: task_d_evolve_api_ts/notificationService.ts
export class NotificationService {
  sentNotifications: string[] = [];

  send(user: string, message: string): void {
    /** Send a notification to a user. */
    const notification = `To ${user}: ${message}`;
    this.sentNotifications.push(notification);
    console.log(notification);
  }

  getHistory(): string[] {
    /** Return the list of all sent notifications. */
    return [...this.sentNotifications];
  }
}
